File: HanJie.CSLCN.Models/Dtos/WikiPassageDto.cs
using HanJie.CSLCN.Models.DataModels;
using System;
using System.Collections.Generic;
using System.Text;

namespace HanJie.CSLCN.Models.Dtos
{
    public class WikiPassageDto : BaseDtoModel<WikiPassageDto, WikiPassage>
    {
        public string Title { get; set; }
        public string Content { get; set; }
        public string RoutePath { get; set; }
        public List<WikiPassageAnchorTitleDto> AnchorTitles { get; set; }
        public UserInfoDto LastModifyUser { get; set; }
        public List<UserInfoDto> MainAuthors { get; set; }
        public List<UserInfoDto> CoAuthors { get; set; }
        public int CategoryId { get; set; }
        public string CategoryUrl { get; set; }
        public List<BreadCrumbDto> BreadCrumbs { get; set; }
        public UserInfoDto EditingUser { get; set; }
        public int TotalViewsCount { get; set; }
    }
}

File: HanJie.CSLCN.WebApp/ClientApp/src/app/models/wiki-passage-dto.ts
import { AnchorTitle } from './anchor-title';
import { UserInfoDto } from './user-info-dto';
import { BreadCrumbDto } from './bread-crumb';
import { WikiPassageCommentDto } from './wiki-passage-comment-dto';

export class WikiPassageDto {
    id: number;
    title: string;
    content: string;
    routePath: string;
    createDate: string;
    lastModifyUser: UserInfoDto;
    anchorTitles: AnchorTitle[];
    mainAuthors: UserInfoDto[];
    coAuthors: UserInfoDto[];
    categoryId: number;
    categoryUrl: string;
    breadCrumbs: BreadCrumbDto[];
    childPageBreadCrumbs: BreadCrumbDto[];
    editingUser: UserInfoDto;
    totalViewsCount: number;
    lastModifyDate: string;
    comments:WikiPassageCommentDto[];
}
